packages/core/src/Renderer.ts

```diff
@@ -376,14 +375,30 @@ export class Renderer extends AbstractRenderer
      * Renders the object to its WebGL view
      *
      * @param displayObject - The object to be rendered.
-     * @param [renderTexture] - The render texture to render to.
-     * @param [clear=true] - Should the canvas be cleared before the new render.
-     * @param [transform] - A transform to apply to the render texture before rendering.
-     * @param [skipUpdateTransform=false] - Should we skip the update transform pass?
+     * @param {object} [options] - Object to use for render options.
+     * @param {PIXI.RenderTexture} [options.renderTexture] - The render texture to render to.
+     * @param {boolean} [options.clear=true] - Should the canvas be cleared before the new render.
+     * @param {PIXI.Matrix} [options.transform] - A transform to apply to the render texture before rendering.
+     * @param {boolean} [options.skipUpdateTransform=false] - Should we skip the update transform pass?
      */
-    render(displayObject: IRenderableObject, renderTexture?: RenderTexture,
-        clear?: boolean, transform?: Matrix, skipUpdateTransform?: boolean): void
+    render(displayObject: IRenderableObject, options: IRendererRenderOptions = {}, ...rest:any[]): void
     {
+        if (options && 'baseTexture' in options)
```

Comment:
It does seem a bit weird that we're assuming that having a "baseTexture" property means that `options` is a Texture, and therefore the legacy signature should be used. I would suggest using `instanceof Texture` instead. That means developers won't have to sanitize their options of a potential "baseTexture".

Reply:
Yeah, I just fixed this.